src/effects/VST3/VST3Effect.cpp

```diff
@@ -403,6 +404,9 @@ bool VST3Effect::LoadSettings(
       } while(parms.GetNextEntry(key, index));
    }
 
+   if(mPlainUI != nullptr)
+      mPlainUI->ReloadParameters();
```

Comment:
I see that in the last commit of this pull request, `ReloadParameters` gets some uses but `UpdateParameter` is still not used anywhere.

Reply:
These four new calls to `ReloadParameter` will become just one call inside `TransferDataToWindow` when Pietro rebases his work for VST3 effect settings onto this branch.

I want the various settings-loading functions not to have the added responsibility to update UI but that's not a requirement to approve this pull request.